test/03-encryption-key.js

```diff
@@ -0,0 +1,30 @@
+const tape = require('tape')
+const crypto = require('crypto')
+const bfe = require('../')
+
+tape('03 encryption key', function (t) {
+  const key = crypto.randomBytes(32)
+
+  const values = [
+    'ssb:diffie-hellman/curve25519/' +
```

Comment:
Note that if you intend to use this as a real SSB URI, please update https://github.com/ssb-ngi-pointer/ssb-uri-spec too, because SSB URIs are split into Canonical and Experimental (which acts as an `any`), and this diffie-hellman one looks like a canonical.

Reply:
Well we're in a pickle because I need the bfe.jsin to be merged into the spec repo before I can tidily propose renaming type diffie-hellman and also propose adding a new format to that, before I can modify the types and formats in this test to teddies on those new names and definitive which I actually want!

Reply:
No hurry with it, no deadlock, I'm just reminding us all about this before it gets forgotten. I can open an issue in fact

Reply:
https://github.com/ssb-ngi-pointer/ssb-uri-spec/issues/5